content/company/strategy/index.md

```diff
@@ -59,6 +59,29 @@ Additionally, these are our [product design principles](../../product-engineerin
 - If your organization is getting value from Sourcegraph with a lot of users, our [pricing](https://about.sourcegraph.com/pricing) is designed so that we earn money from you. This lets us invest in improving our product.
 - All users at a given customer are on the same pricing tier. This is simpler than having users at different tiers and encourages us to build things that are broadly valuable.
 
+## Use cases
+
+Use cases help us align the company around the most important value drivers in our product. A good use case should be:
+
+1. TBD
+2. TBD
+3. TBD
+
+Use cases are used by all the departments at source graph to do different things:
+
+- Product (which includes product management, design, and technical writing) relies on them to help make prioritization decisions because they represent proven value drivers for our customers. The use cases also provide a lens through which the team can evaluate design and workflows within and across product areas, and help they improve outcomes by ensuring that what we build aligns with a coordinated messaging strategy.
```

Comment:
```suggestion
- [Product](../../product/index.md) (which includes [product management](../../product/product_management), [design](../../product/design), and [technical writing](../../product/technical_writing)) relies on them to help make prioritization decisions because they represent proven value drivers for our customers. The use cases also provide a lens through which the team can evaluate design and workflows within and across product areas, and help they improve outcomes by ensuring that what we build aligns with a coordinated messaging strategy.
```